cmd/coder/main.go

```diff
@@ -36,7 +36,7 @@ func (r *rootCmd) Subcommands() []cli.Command {
 		&logoutCmd{},
 		&shellCmd{},
 		&syncCmd{},
-		&urlCmd{},
+		&urlsCmd{},
```

Comment:
Whoops, I hda branched off of @anthonyshull upcoming 1.9.0 devurl branch. I'll re-up with a clean delta.